node-gui/src/main_window/main_menu.rs

```diff
@@ -130,7 +130,41 @@ fn make_menu_file<'a>(wallet_mode: WalletMode) -> Item<'a, MenuMessage, Theme, i
                     // TODO: enable setting when needed
                     // menu_item("Settings", MenuMessage::NoOp),
                     menu_item("Exit", MenuMessage::Exit),
-                ]
+                ];
+                #[cfg(feature = "trezor")]
+                {
+                    let mut menu = menu;
+                    menu.insert(
+                        1,
+                        menu_item(
+                            "Create new Trezor wallet",
+                            MenuMessage::CreateNewWallet {
+                                wallet_type: WalletType::Trezor,
+                            },
+                        ),
+                    );
```

Comment:
removed the unnecessary generation of a mnemonic in the trezor case. 
For the Create vs Recover I am not sure we should remove them, the behavior for skipping the scanning has been there for a long time, also most users will be using the Create as they are making new wallets. Also doing Recover on the current Gui wallet is very slow, because of the way the events are handled.